product_manufacturer/__openerp__.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Products Manufacturers',
+    'version': '1.0',
+    'author': 'OpenERP SA',
+    'contributors': ['Acysos SL <info@acysos.com>'],
+    'category': 'Purchase Management',
+    'depends': ['product'],
+    'demo': [],
+    'description': """
+A module that adds manufacturers and attributes on the product form.
```

Comment:
In v8, you can put this text in a separated file called README.rst, and thus it can be parsed also by code view in GitHub. See an example here: https://github.com/odoomrp/odoomrp-wip/tree/8.0/stock_purchase_unit